From: Simon McVittie <smcv@debian.org>
Date: Mon, 9 Oct 2017 09:22:12 +0100
Subject: icu_sources_data.py: Decouple from Mozilla build system

mozpack.path is a wrapper around os.path that normalizes path
separators on Windows, but on Unix we only have one path separator
so there's nothing to normalize. Avoid needing to import all of it.
---
 intl/icu_sources_data.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: mozjs-60.8.0/intl/icu_sources_data.py
===================================================================
--- mozjs-60.8.0.orig/intl/icu_sources_data.py
+++ mozjs-60.8.0/intl/icu_sources_data.py
@@ -19,7 +19,9 @@ import subprocess
 import sys
 import tempfile
 
-from mozpack import path as mozpath
+# Close enough
+import os.path as mozpath
+mozpath.normsep = lambda p: p
 
 
 def find_source_file(dir, filename):
